docs/.project/ALPHA_RELEASE.md

```diff
@@ -41,13 +42,12 @@ _To be done after code freeze and prior to the release day to prepare and build
     * [ ] Works on Mac
     * [ ] Works on Windows
 * [ ] trigger QA for fuzzy testing
-* [ ] write blog post for [blog.camunda.org](https://github.com/camunda/blog.camunda.org), __do not publish until released version is available for download on [camunda.org](https://camunda.org/release/camunda-modeler/)__
+* [ ] write blog post for [blog.camunda.org](https://camunda.com/blog/) according to [this process](https://confluence.camunda.com/pages/viewpage.action?spaceKey=MAR&title=Your+Content+on+the+Camunda+Blog) by copy and pasting [this template](https://docs.google.com/document/d/18aXWFBdbET97EZ_HO1DEVLVsYP-UQEabumSzQdeRbDw/edit?usp=sharing), writing a draft version of the blog post and sending it to Charley Mann and Christopher Rogers for review. Charley will coordinate the publishing.
```

Comment:
If we don't have the blog post content not inside the GitHub repository anymore, should we create a Google Drive where we collect the Docs files?

Reply:
You mean a Google Drive for just the Modeling blog posts?

Good idea :+1: 

(Although I would of course prefer to stay with GitHub as central repo incl. blog posts, so maybe we find a way to get back to that in the future)

Reply:
👍